opentreemap/treemap/js/src/CanopyFilterControl.js

```diff
@@ -0,0 +1,56 @@
+"use strict";
+
+var $ = require('jquery'),
+    _ = require('lodash'),
+    L = require('leaflet'),
+    Bacon = require('baconjs');
+
+require('ionRangeSlider');
+
+var canopyFilterTmpl = _.template($('#canopy-filter-tmpl').html());
+
+var CanopyFilterControl = L.Control.extend({
+    options: {
+        position: 'bottomleft'
+    },
+
+    initialize: function() {
+        this._changeBus = new Bacon.Bus();
+        this.tilerArgsProp = this._changeBus.toProperty({
+            category: 'Municipality',
```

Comment:
Instead of hard coding a value, the `category` should come from the instance config. The canopy control behavior is not affected by the value of `category`, so we may want to separate our concerns and have the stream consumer append it.

```javascript
filterControl.tilerArgsProp
             .debounce(1000)
             .map(function(args) {
               args.category = config.category;
               return args;
             })
             .onValue(...)